include/_kernel.h

```diff
@@ -32,6 +32,30 @@
 #endif
 #include "_kernel_c.h"
 
+/* If the -cl-std build option is not specified, the highest OpenCL C 1.x
+ * language version supported by each device is used as the version of
+ * OpenCL C when compiling the program for each device.
```

Comment:
Yep, it is OK. That quote is directly from: https://www.khronos.org/registry/cl/sdk/2.0/docs/man/xhtml/preprocessorDirectives.html

Now that i think of it, i took care of this particular thing in pocl_llvm_api.cc so the ifndef is never going to happen.